CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Conjur CLI now uses the custom `rest_client_options` new to `conjur-api` 5.3.5.
```

Comment:
Let's add the fact that the usage is implicit/internal, since there isn't a public facing CLI specific mechanism for using the rest_client_options.

Reply:
Maybe like this:
```suggestion
- Conjur CLI internally uses `rest_client_options` new to `conjur-api` 5.3.5.
```

Reply:
I think this should also be under the `Changed` section